### DISCUSSION TITLE
always show a empty repo on notification page

### DISCUSSION THREAD

**Discussion by zirain** [community/community#127291] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Not sure if this's a bug, there's always show a repo(istio/istio.io) on my notification page when there's no notification from it.

<img width="1043" alt="image" src="https://github.com/community/community/assets/4354057/99782925-d77c-4833-8180-39cd109ce46f">
<img width="804" alt="image" src="https://github.com/community/community/assets/4354057/ef63ba30-da23-4a57-91b3-680672813b7a">

**Comment by InboraStudio** [comment]
It's not necessarily a bug, but an issue with notification settings or activity on the repository. Check your notification settings for the repository to ensure they're configured correctly.
